mxcube3/core/components/queue.py

```diff
@@ -1382,6 +1383,9 @@ def _create_queue_entry(self, task, task_name):
         Return:
             (tuple): (model, entry)
         """
+        if not task["parameters"]["osc_range"]:
```

Comment:
I dont understand this 🧐...

`osc_range` comes with a `null` value? Should not this be fixed in the ui?

Reply:
Yes, you are indeed right that it seems a bit odd. However now a days and especially with SSX oscillation range might not be set or used at all hence its `Null`. We could perhaps set it to 0 instead. There is some thought work to do and its to remove the "composite collection" dictionary that we are using for the "traditional" oscillation experiments.